utils/docker/examples/client/README.md

```diff
@@ -215,3 +215,87 @@ $ dfuse --mountpoint="/home/<DAOS_CLIENT_UNAME>/mnt" --pool=tank --container=pos
+- `DAOS_DOCKER_IMAGE_TAG`: Tag identifier of the DAOS client docker image (default "rocky8.6")
+- `DAOS_CLIENT_UNAME`: User name of the client (e.g. "foo")
+- `DAOS_CLIENT_GNAME`: Group name of the client (e.g., "bar")
+
+When the environment file has been properly filled, the docker image could be created thanks to the
+following commands:
+```bash
+docker compose --file utils/docker/examples/client/docker-compose.daos_client_agent.gathered.yml build daos_client_agent
+```
+
+### Running DAOS Docker Containers
+
+This section presents two different way for running some relevant use cases with a docker image
+build according to the previous section.  For both methods, a tarball (i.e. `tar` archive compressed
+with `xz`) of the DAOS certificate files should be created when the DAOS authentication is enabled.
+However, it is not managed in the same way with both solutions.
+
+#### Running DAOS Docker Images with Docker Compose
+
+For using Docker Compose the tarball of the certificates file path readable by all users and its
+file path defined in the following variable of the docker environment file
+`utils/docker/examples/client/.env`:
+- `DAOS_AGENT_CERTS_TXZ`: tarball containing the DAOS certificated needed by the DAOS agent
+  (e.g. "secrets/daos\_agent-certs.txz")
+- `DAOS_CLIENT_UID`: User id of the client (e.g.,  "666")
+- `DAOS_CLIENT_GID`: Group id of the client (e.g., "999")
+
+When the environment file has been properly filled, then an application such as `daos pool autotest`
+could be run in the following way:
+```bash
+docker compose --file utils/docker/examples/client/docker-compose.daos_client_agent.gathered.yml run --rm daos_client_agent
+$ daos pool autotest <POOL ID>
+```
+
+With the same prerequites as for the previous section, the [fio](https://fio.readthedocs.io/) file
+system benchmark tool could be run thanks to the following commands:
+```bash
+docker compose --file utils/docker/examples/client/docker-compose.daos_client_agent.gathered.yml run --rm daos_client_agent
+$ mkdir -p "/home/<DAOS_CLIENT_UNAME>/mnt"
+$ daos container create --type=posix --label=posix-fs tank
+$ dfuse --mountpoint="/home/<DAOS_CLIENT_UNAME>/mnt" --pool=tank --container=posix-fs
+$ df --human-readable --type=fuse.daos
+$ fio --name=random-write --ioengine=pvsync --rw=randwrite --bs=4k --size=128M --nrfiles=4 --numjobs=8 --iodepth=16 --runtime=60 --time_based --direct=1 --buffered=0 --randrepeat=0 --norandommap --refill_buffers --group_reporting --directory="/home/<DAOS_CLIENT_UNAME>/mnt"
+```
+
+#### Running DAOS Docker Images with Docker Stack
+
+With Docker Stack the tarball of the certificates are managed as [Docker
+Secret](https://docs.docker.com/engine/swarm/secrets/).  Docker Secret is a swarm service allowing
+to securely store and access blob of data.  Recording a tarball containing the DAOS agent
+certificates could be done in the following way:
+```bash
+docker swarm init
+docker secret create daos_agent-certs <TARBALL PATH>
```

Comment:
Do you want to use <TARBALL PATH> or <DAOS_AGENT_CERTS_TXZ>? It's up to you.

Reply:
- [ ] Fix file name

Reply:
Fixed with commit 94325c116181e96f9685662051b275f0c3fc5bd5